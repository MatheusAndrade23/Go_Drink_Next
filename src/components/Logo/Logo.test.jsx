import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { Logo } from '.';

describe('<Logo />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Logo text="GODRINK" smallText="GD" />);
    expect(container).toMatchSnapshot();
  });
});
