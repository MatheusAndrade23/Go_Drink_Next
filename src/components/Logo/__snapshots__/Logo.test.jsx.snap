// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render 1`] = `
.c0 {
  color: #00ADB5;
  font-weight: bold;
  text-align: center;
  font-size: 32px;
}

.c0 p {
  font-family: 'Montserrat',sans-serif;
}

.c1 {
  font-family: 'Montserrat',sans-serif;
  display: none;
}

@media (max-width:768px) {
  .c0 p {
    display: none;
  }
}

@media (max-width:420px) {
  .c0 p {
    display: block;
  }
}

@media (max-width:768px) {
  .c1 {
    min-width: 100px;
    display: block;
  }
}

@media (max-width:420px) {
  .c1 {
    display: none;
  }
}

<div>
  <a
    class="c0"
    href="/"
  >
    <p>
      GODRINK
    </p>
    <span
      class="c1"
    >
      GD
    </span>
  </a>
</div>
`;
